weatherbot/map.py

```diff
@@ -0,0 +1,18 @@
+import folium
+
+from weatherbot.weather import get_weather
+
+
+def get_coords(city):
+    coords_list = []
+    response = get_weather(city)
+    coords = response["coord"]
+    coords_list.append(coords['lat'])
+    coords_list.append(coords['lon'])
+    return coords_list
```

Comment:
I'd rewrite it as `return [coords['lat'], coords['lon']]`

Reply:
Okay, I will fix it.